tests/test_default_gaussian.py

```diff
@@ -534,6 +534,53 @@ def test_reduced_state(self):
         self.assertAllAlmostEqual(res[0], expected[0], delta=self.tol)
         self.assertAllAlmostEqual(res[1], expected[1], delta=self.tol)
 
+    def test_samples_coherent(self):
+        """Test that sampled values are calculated correctly by comparing with first and second moments"""
+        self.logTestName()
+        n_samples = 1000000
+        dev = qml.device('default.gaussian', wires=1, hbar=hbar)
+
+        # test correct mean and variance for Homodyne P measurement
+        alpha = 0.324-0.59j
+        dev.apply('CoherentState', wires=[0], par=[alpha])
+        samples = dev.sample('P', [0], [], n_samples)
+        mean = samples.mean()
+        std = samples.var()
+
+        self.assertAlmostEqual(mean, alpha.imag*np.sqrt(2*hbar), delta=1000*self.tol)
+        self.assertAlmostEqual(std, hbar/2, delta=1000*self.tol)
+
+        # test correct mean and variance for Homodyne measurement
+        samples = dev.sample('P', [0], [], n_samples)
+        mean = samples.mean()
+        std = samples.var()
+
+        self.assertAlmostEqual(mean, alpha.imag*np.sqrt(2*hbar), delta=1000*self.tol)
+        self.assertAlmostEqual(std, hbar/2, delta=1000*self.tol)
+
+    def test_samples_squeezed(self):
+        """Test that sampled values are calculated correctly by comparing with first and second moments"""

```

Comment:
```suggestion
        """Test that sampled values are calculated correctly for a squeezed state by comparing with first and second moments"""
```
I think it is just nicer to have no copy-pasted docstrings.